project-chip/connectedhomeip
# Implement video player Binding and ACL functionality

#### Problem
1. BLE commissioning of tv-app does not work when commissioner is enabled
2. ContentLaunch from tv-casting-app broken due to new ACL enforcement

### Change overview
1. Add management of transport to BLE code
2. Add ACLs for casting app

#### Testing
How was this tested? (at least one bullet point required)
1. Verify BLE commissioning of tv-app works
2. Verify tv-casting-app call to contentLauncher succeeds
